src/actions/publish.js

```diff
@@ -85,13 +85,17 @@ export function uploadFailed(error: mixed): Action {
 
 // This function stores information about the published profile, depending on
 // various states. Especially it handles the case that we sanitized part of the
-// profile.
+// profile. The sanitized information is passed along because it can be costly
+// to rerun in case the selectors have been invalidated.
+// Note that the returned promise won't ever be rejected, all errors are handled
+// here.
 async function storeJustPublishedProfileData(
   profileToken: string,
   jwtToken: string | null,
-  state: State
+  sanitizedInformation: *,
```

Comment:
I think this would be better as `sanitizedInformation: SanitizeProfileResult`.

Reply:
I didn't like the name of this type, and it's not in a "types" file but a "logic" file, that's why I didn't want to import it, this felt a bit "pasta". In this case the function is used in the same file and I double-checked that Flow checks errors. Actually the behavior with errors is slightly better without specifying `: *` (with `*` flow infers the type from the funvtion body, but without `*` flow infers the type from the caller, so it uses `SanitizeProfileResult`), so I removed it.